imports/startup/server/index.coffee

```diff
@@ -168,6 +168,43 @@ Meteor.methods
       """
     makeRequest(query)
 
+  'getTrendingInfectiousAgents': (date, date2, days) ->
```

Comment:
One of these parameters is redundant. The function should be able to compute days from the difference of the two dates, or it could just take a time period parameter and compute the dates.